setup.py

```diff
@@ -71,11 +71,12 @@
     url="https://github.com/ApeWorX/ape-hardhat",
     include_package_data=True,
     install_requires=[
-        "eth-ape>=0.6.9,<0.7",
+        "eth-ape>=0.6.11,<0.7",
         "evm-trace",  # Use same version as eth-ape
         "hexbytes",  # Use same version as eth-ape
         "web3",  # Use same version as eth-ape
         "chompjs>=1.1.9,<2",  # To help parse hardhat files
+        "yarl",  # Use same version as ape
```

Comment:
I dont see this in `eth-ape`, might need to specify the version range here and remove this comment